app/components/alerts/alerts.component.spec.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
+
```

Comment:
Also, I think I do like having the test co-located with the file.  If we improve the structure so that related bits are all together, it makes it very clear when something isn't properly tested.